src/Chapter02/Listing02.21.WorkingWithJSONUsingRawStringLiterals.cs

```diff
@@ -0,0 +1,53 @@
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter02.Listing02_21
+{
+    #region INCLUDE
+    // The using directives allow you to drop the namespace
+    #region HIGHLIGHT
+    #endregion HIGHLIGHT
+    public class Program
+    {
+        public static void Main()
+        {
+            string firstName = "Forest";
+
+            string lastName = """Gump"""; // Single-line raw string literal.
+
+            string greeting = // Single-line raw string literal with interpolation.
+
+            $"""Hello, I'm {firstName}. {firstName} {lastName}""";
```

Comment:
Do we want these comments here or should they be one line above? And then the string itself can be moved up? A few of these in this document

Reply:
We can't have lines longer than 72 characters, so this should be on the prior line.